components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
```

Comment:
I missed this comment before landing this PR but there are some other comments I need to add via #20, so I'll make a separate PR with all of those.